books/free-programming-books-bn.md

```diff
@@ -53,7 +53,7 @@
 * [Pattern Printing in C](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oTxQhx2kejYmQn6qtRULCD) - Anisul Islam
 
 
-<h3 id="cpp">C++</h3>
+### C++
```

Comment:
See how it's addresed in other files to be synced with index entry: https://github.com/EbookFoundation/free-programming-books/pull/6621/files#R8

```markdown
* <a href="#cpp">C++</a>
```

Headings with only one letter and symbols makes some problem with rendering bugs across GitHub auto ToC, GMF markdown preview and the HTML rendered by GitHub pages.

```markdown
### <a name="cpp"></a>C++
```

Reply:
Now link doesn't work.

Needs change `href="cpp"` too to have a working Index-To-Section linking.

Check it clicking in `C++` Index item.